Examples/Algorithms/Vertexing/src/IterativeVertexFinderAlgorithm.cpp

```diff
@@ -50,21 +47,22 @@ ActsExamples::ProcessCode ActsExamples::IterativeVertexFinderAlgorithm::execute(
     const ActsExamples::AlgorithmContext& ctx) const {
   // retrieve input tracks and convert into the expected format
 
-  auto [inputTrackParameters, inputTrackPointers] =
-      makeParameterContainers(ctx, m_inputTrackParameters, m_inputTrajectories);
+  const auto& inputTrackParameters = m_inputTrackParameters(ctx);
+  auto inputTrackPointers =
```

Comment:
Could you add a comment here that the vertexing assumes it gets stable pointers into parameters objects, and that we intend to change this?